controllers/grants.go

```diff
@@ -128,16 +128,25 @@ func RedeemGrants(w http.ResponseWriter, r *http.Request) {
 
 	txInfo, err := req.Redeem(r.Context())
 	if err != nil {
+		if err, ok := err.(*grant.RedeemError); ok {
+			fmt.Println(err)
```

Comment:
general comment, I saw you used `Println` and other `fmt` methods in a couple of places should use `log` (actually alias for `logrus` methods instead